### DISCUSSION TITLE
. health and . resolve picked up in token action HUD

### DISCUSSION THREAD

**Discussion by SwedishRabbit** [fvtt-fria-ligan/forbidden-lands-foundry-vtt#392] [General]
Hello, it seems like .health and . resolve exists in the actor sheet. It gets actions in the token action HUD. These actions are not used. Is there anyway to delete these two attributes without affecting the actor sheet or any downstream functions?

**Comment by aMediocreDad** [comment]
This is a problem that needs to be dealt with by Token Action HUD. It needs to check whether the "health"/"resolve" optional rules are enabled and if not remove them from the HUD.

**Reply by SwedishRabbit** [reply]
Okay, thanks for your answer!
